Assignment.java

```diff
@@ -0,0 +1,71 @@
+package package1;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class Assignment {
+	
+	WebDriver driver;
+
+	@Before
```

Comment:
and this should be @beforeEach

Reply:
Junit which has been currently used does not give me an option for beforeeach